config/secrets.yml

```diff
@@ -12,10 +12,13 @@
 
 development:
   secret_key_base: 9fa9a1fdcda359a1cdce2d1e45b2799397acc5d5b962c4aa4c72207330fd3ce26e6acec15818d419ddf0aa05fa2f080654d435473858afbd5767dc11911e67ba
+  google_client_id: 757656209931-n8t7v63pfl907i02rt96brrfi4nil4ak
+  google_client_secret: AYt1TuBizF1MF1ZJHDxAKmjq
 
 test:
   secret_key_base: 0f795c1ada24ff4780d43bb2ad7896eee86616728ccb7e5b0f5b924149991e73ac51140e3e35b69bfd4df106bda484a797c2ba3c7501e24cb29b0e159ccef4e7
-
+  google_client_id: 757656209931-n8t7v63pfl907i02rt96brrfi4nil4ak
+  google_client_secret: AYt1TuBizF1MF1ZJHDxAKmjq
```

Comment:
@mollybrown any ideas? In the meantime, I'll gitignore this and re-push this branch.